commands/team_e2e_test.go

```diff
@@ -51,3 +53,41 @@ func (s *MmctlE2ETestSuite) TestRenameTeamCmdF() {
 		s.Equal("Cannot rename team '"+s.th.BasicTeam.Name+"', error : : You do not have the appropriate permissions., ", err.Error())
 	})
 }
+
+func (s *MmctlE2ETestSuite) TestArchiveTeamsCmd() {
+	s.SetupTestHelper().InitBasic()
+
+	cmd := &cobra.Command{}
+	cmd.Flags().Bool("confirm", true, "Confirm you really want to archive the team and a DB backup has been performed.")
+
+	s.RunForAllClients("Archive nonexistent team", func(c client.Client) {
+		printer.Clean()
+
+		err := archiveTeamsCmdF(c, cmd, []string{"unknown-team"})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 0)
+		s.Require().Len(printer.GetErrorLines(), 1)
+		s.Require().Equal("Unable to find team 'unknown-team'", printer.GetErrorLines()[0])
+	})
+
+	s.RunForSystemAdminAndLocal("Archive basic team", func(c client.Client) {
+		printer.Clean()
+
+		err := archiveTeamsCmdF(c, cmd, []string{s.th.BasicTeam.Name})
+		s.Require().Nil(err)
+		s.Require().Len(printer.GetLines(), 1)
+		team := printer.GetLines()[0].(*model.Team)
+		s.Require().Equal(s.th.BasicTeam.Name, team.Name)
+		s.Require().Len(printer.GetErrorLines(), 0)
+	})
```

Comment:
We should retrieve the team from the database using `s.th.App` and assert that after the command's execution it is now archived. After that, the team should be unarchived so the rest of the tests start from the right state.

I'm thinking on something like:

```go
basicTeam, err := s.th.App.GetTeam(s.th.BasicTeam.Id)
s.Require().Nil(err)
s.Require().NonZero(basicTeam.DeleteAt)

err := s.th.App.RestoreTeam(s.th.BasicTeam.Id)
s.Require().Nil(err)
```